molgenis-core-ui/src/main/frontend/src/scss/bootstrap-molgenis-blue.scss

```diff
@@ -78,6 +78,8 @@ $btn-border-radius-lg: 0px;
 
 // Navbar
 $navbar-light-color: $main-blue;
+$navbar-light-hover-color: #333333;
```

Comment:
suggest to replace (#333333) with named var and see if relative val is more appropriate.